hack/python-sdk/update_release_version_helper.py

```diff
@@ -33,6 +33,7 @@
     "python/pmmlserver/pyproject.toml",
     "python/sklearnserver/pyproject.toml",
     "python/xgbserver/pyproject.toml",
```

Comment:
This file too can be removed. Instead we are using [this script](https://github.com/kserve/kserve/blob/master/hack/python-release.sh).